tests/sorter.test.js

```diff
@@ -0,0 +1,33 @@
+const response = require('../fixtures/bar_results.json');
+const Sorter = require('../src/sorter')
+
+describe('Sorter', () => {
+  const sorter = new Sorter;
+
+  describe('#topFiveBars', () => {
+    const results = response.results;
+    const topFiveResults = sorter.topFiveBars(results);
+
+    it('filters results to rating over four', () => {
+      aboveFourRating = topFiveResults.every( bar => bar.rating > 4 );
+
+      expect(aboveFourRating).toEqual(true);
+    })
+
+    it('adds a popular rating to each bar', () => {
+      const leastRecommended = topFiveResults[0];
+
+      expect(leastRecommended.popularRating).toEqual(53.2)
```

Comment:
LeastRecommended is the lowest scoring bar right? Maybe let's change popularRating to popularityRating instead so it's a bit clearer that it's not like leastRecommended but popular.